Automattic/jetpack
# Jetpack CLI: Add new subcommand and script to rsync plugin files to remote destinations.

Sometimes I find myself not wanting to use Docker and tunneling. I've also sometimes wanted to develop against a JN site... this makes it pretty easy. 

![image](https://user-images.githubusercontent.com/7129409/193824689-c430f00f-3edd-48fb-9884-91b4a8239734.png)

#### Changes proposed in this Pull Request:

New `jetpack rsync` command for things in projects/plugins. 

#### Jetpack product discussion


Gives more options for developers. 

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:





- Spin up a JN site without any plugins pre-installed. 
And use after building one: 
- `jetpack build plugins/social`
- `jetpack rsync` => follow the commands. 
- `jetpack rsync social` => follow the commands
- After following all the prompts, activate the plugin. It should be working as expected. 

Also try after building with `--production`

Also try passing the commands directly to it. Example: 
- `jetpack rsync jetpack user@your.server.example.com:/home/path/to/wp-content/plugins`

Also try removing things from the config after setting it 
- `jetpack rsync --config`
